docs/topics/api/accounts.rst

```diff
@@ -37,6 +37,8 @@ A developer is defined as a user who is listed as a developer or owner of one or
     :>json string|null picture_type: the image type (only 'image/png' is supported) if a user defined photo has been provided, or none if no photo has been provided.
     :>json boolean is_addon_developer: The user has developed and listed add-ons on this website.
     :>json boolean is_artist: The user has developed and listed themes on this website.
+    :>json boolean has_anonymous_display_name: The user hasn't chosen a name or username.
```

Comment:
I would rephrase this to "The user has chosen neither a name nor a username." I know it's more verbose but it should be clear it's about having chosen neither, not having chosen neither or only one.

Reply:
https://github.com/mozilla/addons-server/pull/6426/commits/5478817743bc9827f0036a08da376040c2eb1def